lib/puppet/pops/issues.rb

```diff
@@ -368,6 +368,10 @@ def self.hard_issue(issue_code, *args, &block)
     "Resource Defaults are not virtualizable"
   end
 
+  CLASS_NOT_VIRTUALIZEABLE = issue :CLASS_NOT_VIRTUALIZEABLE do
```

Comment:
I think the spelling should be CLASS_NOT_VIRTUALIZABLE to be consistent with its own text (also, "virtualizable" gives me 22.200 hits on Google. "virtualizeable" just 1.420).